src/components/MoneyRequestConfirmationList.js

```diff
@@ -171,7 +171,7 @@ function MoneyRequestConfirmationList(props) {
     const {unit, rate, currency} = props.mileageRate;
     const distance = lodashGet(transaction, 'routes.route0.distance', 0);
     const shouldCalculateDistanceAmount = props.isDistanceRequest && props.iouAmount === 0;
-    const shouldCategoryEditable = !_.isEmpty(props.policyCategories) && !props.isDistanceRequest;
+    const shouldCategoryEditable = !_.isEmpty(props.policyCategories);
```

Comment:
NAB: I know you didn't create this variable, but maybe we can update it to use correct grammar while you're here. 

```suggestion
    const shouldCategoryBeEditable = !_.isEmpty(props.policyCategories);
```

Reply:
Agree. Will update.